src/entities/user_reset_password_request.entity.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+  ManyToOne,
+  BeforeInsert,
+  JoinColumn,
+} from "typeorm";
+import { User } from "./user.entity";
+import { Random } from "../utils/random";
+
+@Entity({ name: "users_reset_password_requests" })
+export class UserResetPasswordRequest {
+  @PrimaryGeneratedColumn()
+  id: string;
+
+  @Column({ name: "expires_at", nullable: true })
+  expiresAt: Date;
+
+  @Column({ name: "resendable_at", nullable: true })
+  resendableAt: Date;
+
+  @Column({ name: "key", nullable: true })
```

Comment:
"key" shouldn't be nullable since every reset password request should have a key.